front_end/generated/protocol.d.ts

```diff
@@ -12993,6 +12993,19 @@ declare namespace Protocol {
       System = 'system',
     }
 
+    /**
+     * COHERENT_BEGIN
+     */
+    export interface TraceCategoryDefinition {
+      name: string;
+      categoryString: string;
+    }
+
+    export interface GetTraceSystemsAndLevelsResponse extends ProtocolResponseWithError {
+      systems: TraceCategoryDefinition[];
+      levels: TraceCategoryDefinition[];
+    }
```

Comment:
```suggestion
    // COHERENT BEGIN
    export interface TraceCategoryDefinition {
      name: string;
      categoryString: string;
    }

    export interface GetTraceSystemsAndLevelsResponse extends ProtocolResponseWithError {
      systems: TraceCategoryDefinition[];
      levels: TraceCategoryDefinition[];
    }
    // COHERENT END
```

Reply:
this is a generated file. All `COHERENT BEGIN` / `COHERENT END` comments for changes responsible for this are in `browser_protocol.pdl`